sdks/go/pkg/beam/core/runtime/exec/optimized/inputs.go

```diff
@@ -5900,9 +5791,8 @@ func (v *iterNative) readInt16String(key *int16, value *string) bool {
 		}
 		panic(fmt.Sprintf("broken stream: %v", err))
 	}
-
```

Comment:
Why did this file change so much?

Reply:
It's to work around https://issues.apache.org/jira/browse/BEAM-3580. The join example happens to use string values, which were broken before.

Note that all the .go files are generated from the .tmpl files.